src/hev-main.h

```diff
@@ -28,7 +28,7 @@ extern "C" {
  *
  * Since: 2.4.6
  */
-int hev_socks5_tunnel_main (const char *config_path, int tun_fd);
+int hev_socks5_tunnel_main (const char *config_path, const unsigned char *config_str, unsigned int config_len, int tun_fd);
```

Comment:
Could you add a new method/function to avoid break the current API? Thanks.

Reply:
@heiher added `hev_socks5_tunnel_main2()` awkward name but it's either long or that.. :)

Reply:
Thanks.